crates/revm/Cargo.toml

```diff
@@ -74,11 +74,11 @@ alloydb = ["database/alloydb"]
 serde-json = ["serde", "inspector/serde-json"]
 
 dev = [
-    "memory_limit",
-    "optional_balance_check",
-    "optional_block_gas_limit",
-    "optional_eip3607",
-    "optional_no_base_fee",
+	"memory_limit",
+	"optional_balance_check",
+	"optional_block_gas_limit",
+	"optional_eip3607",
+	"optional_no_base_fee",
```

Comment:
automatic cargo fmt -- will turn off formatting and revert

Reply:
It is better like this, but both works.

Reply:
Oh sorry did not read this, already reverted all of the Cargo.toml changes :)